docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/DnnFeaturizeImage.cs

```diff
@@ -61,6 +62,9 @@ public static void Example()
             // 0.159010679
             // 0.394427955
 
+            // Dispose bitmap image objects.
+            foreach (var img in transformedData.GetColumn<Bitmap>("ImageObject"))
+                img.Dispose();
         }
```

Comment:
What is the purpose of this here?

Reply:
The reason why I ask is, it *might* be easy for someone to misinterpret and think that it is a required step, for some reason.

---
In reply to: [275888856](https://github.com/dotnet/machinelearning/pull/3307#discussion_r275888856) [](ancestors = 275888856)

Reply:
yep, this is not necessary because this is a direct getter that does not go through any other image transform that might lead to the creation of intermediate images.